vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/resources/META-INF/resources/frontend/disableOnClickFunctions.js

```diff
@@ -1,14 +1,6 @@
-function disableOnClickListener({currentTarget: element}) {
-  if (element.disableOnClick) {
-    requestAnimationFrame(() => element.disabled = true);
+document.addEventListener('click', (event) => {
```

Comment:
Out of curiosity: wouldn't this be executed for every button I'm going to add on a page.. so I could technically end up with 100 listener that do exactly the same on every click? Could this also create a memory problem with the exact same reasoning because it's not removed - or am I overthinking?